sdks/java/core/src/main/java/org/apache/beam/sdk/testing/GatherAllPanes.java

```diff
@@ -39,7 +40,7 @@
  * closed the window earlier.
  */
 public class GatherAllPanes<T>
```

Comment:
With the package move, this probably should be made package-private as an implementation detail of `PAssert`